<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info</class>
 <widget class="QMainWindow" name="info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创能机器人配置软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="yuanhongwai">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>远红外摄像头的端口:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="robot_mac">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>135</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="r_mac">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>小车的网络物理地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="wddy">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>温度电压的端口:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duojizhuban_port">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>285</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="wddy_port">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>335</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="maxcharg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>小车电量的最大值:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="appserverip">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>285</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="left_eye_port">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>85</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="yaw">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>yaw</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="host_mac">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>85</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="right_eye">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>右传感器的端口:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="right_eye_port">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>35</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="autocharge_xyz">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>自动充电点坐标x,y,yaw:</string>
    </property>
   </widget>
   <widget class="QLabel" name="left_eye">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>左传感器的端口:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deluge_gun_port">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>135</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="autocharge_yaw">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="y">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="mincharg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>小车电量的最小值:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="autocharge_y">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yuanhongwai_port">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>235</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="x">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="server_ip">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>35</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="autocharge_x">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="duojizhuban">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>舵机主控板的端口:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mincharge">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>185</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cameraip">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>监控摄像头的IP地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fenbushi">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>分布式传感器的端口:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_info">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>480</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="fenbushi_port">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>185</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="camera_ip">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>335</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="s_ip">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>小车的IP地址:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxcharge">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>235</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="h_mac">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>工控机的网络物理地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="app_ip">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>手机端后台服务器的IP地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="deluge_gun">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>水平转动的端口:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionRobot_info"/>
     <addaction name="actionFenbushi"/>
     <addaction name="actionPlanwork"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRobot_info">
   <property name="text">
    <string>Robot_info</string>
   </property>
  </action>
  <action name="actionFenbushi">
   <property name="text">
    <string>Fenbushi</string>
   </property>
  </action>
  <action name="actionPlanwork">
   <property name="text">
    <string>Planwork</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
